python/tvm/autotvm/task/task.py

```diff
@@ -244,8 +345,12 @@ def __call__(self, *args, **kwargs):
     def _default_func(self, *args, **kwargs):
         assert callable(self.fcompute) and callable(self.fschedule)
         out = self.fcompute(*args, **kwargs)
-        arg_bufs = [out] + self._get_inputs(out)
-        s = self.fschedule([out])
+        if GLOBAL_SCOPE.current.tune_subgraph:
+            arg_bufs = _traverse_to_get_io_tensors(out)
+            s = self.fschedule(out)
+        else:
+            arg_bufs = [out] + self._get_inputs(out)
```

Comment:
Note: auf_bufs return by "Line 349", has "inputs" first, and then follows "outputs"